cloudigrade/account/models.py

```diff
@@ -0,0 +1,15 @@
+"""Cloudigrade Account Models."""
+from django.db import models
+
+from util.models import BaseModel
+
+
+class Account(BaseModel):
+    """Account model."""
+
+    account_id = models.DecimalField(
```

Comment:
Will this always be numeric, or do we want a string here to accommodate other clouds?

Reply:
Maybe we should even call the column `aws_account_id`?

Reply:
@ilyavw and I discussed offline… let's just leave this as-is because we're going to have to restructure the whole model anyway when we extend support beyond AWS, and it makes sense to make AWS-specific distinctions at that time.